app/presenters/calendar_presenter.rb

```diff
@@ -0,0 +1,36 @@
+# CalendarPresenter
+# @author Tracy A. McCormick
+# additional methods used for the calendar
+class CalendarPresenter < HoursPresenter
+  # resource_name
+  # @author Tracy A. McCormick
+  # @date 03/28/2017
+  # @param - hash (hash)
+  # returns either the Library name or combined department name & library name
+  def resource_name(hash = {})
+    if hash[:type] == 'department'
```

Comment:
don't use hash as a variable name.  It is a restricted keyword in some ruby frameworks.  If you do prefix it with what it takes.  IE: User_input_hash or get_post_hash or query_string_hash  Be descriptive in your naming, but not too long winded.